spec/controllers/api/v1/accounts/notifications_controller_spec.rb

```diff
@@ -102,4 +102,30 @@
       end
     end
   end
+
+  describe 'GET /api/v1/accounts/{account.id}/notifications/unread_count' do
+    context 'when it is an unauthenticated user' do
+      it 'returns unauthorized' do
+        get "/api/v1/accounts/#{account.id}/notifications/unread_count"
+
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'when it is an authenticated user' do
+      let(:admin) { create(:user, account: account, role: :administrator) }
+      let!(:notification1) { create(:notification, account: account, user: admin) }
```

Comment:
> RSpec/LetSetup: Do not use let! to setup objects not referenced in tests.
We have a rspec rule that prevents using let! if you aren't referencing the variable in tests. 

we can move this create into the `it` block